### PR TITLE
[Bug](Cooldown) fix load balance causing no cooldown replica

### PR DESCRIPTION
**Pull request by ByteYue**
# Proposed changes

Issue Number: close #xxx
Currently cooldown's implementation would compare the replica id of the local replica and cooldown_replica_id inside the cooldown conf, unfortunately the `Tablet::revise_tablet_meta` didn't set the right replica id due to the lack of copy of replica id when using copy constructor of tablet meta. Which would end up tablet meta with replica id equals to 0. And the subsequent comparison would fail, neither would the cooldown operation.
Plus, this pr add some log for better replica tracing when doing cloning.
## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [x] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [x] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 35.12 seconds
 stream load tsv:          475 seconds loaded 74807831229 Bytes, about 150 MB/s
 stream load json:         39 seconds loaded 2358488459 Bytes, about 57 MB/s
 stream load orc:          69 seconds loaded 1101869774 Bytes, about 15 MB/s
 stream load parquet:          28 seconds loaded 861443392 Bytes, about 29 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230211165631_clickbench_pr_94582.html

**Review by dataroaring — APPROVED** [review]
LGTM

File: be/src/olap/task/engine_clone_task.cpp [hunk 1]
```diff
@@ -537,7 +539,8 @@ Status EngineCloneTask::_finish_incremental_clone(Tablet* tablet,
                                                   const TabletMeta& cloned_tablet_meta,
                                                   int64_t committed_version) {
     LOG(INFO) << "begin to finish incremental clone. tablet=" << tablet->full_name()
-              << ", committed_version=" << committed_version;
+              << ", committed_version=" << committed_version
+              << ", cloned_tablet_replica_id=" << cloned_tablet_meta.tablet_id();
```
Comment (platoneko) [inline be/src/olap/task/engine_clone_task.cpp:543]: `tablet_id()` or `replica_id()`?

Comment (ByteYue) [reply be/src/olap/task/engine_clone_task.cpp:543]: fix it in #16644

**Pull request merged**